tests/utils.py

```
@@ -0,0 +1,543 @@
+
+
+class ValidationTestHelper:
+    """Helper functions for validation testing"""
+    
+    @staticmethod
+    def assert_validation_error(func, *args, **kwargs):
+        """Assert that a validation error is raised"""
+        from pydantic import ValidationError
+        try:
+            func(*args, **kwargs)
+            assert False, "Expected ValidationError was not raised"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)